BenadrylAutomation/src/main/java/com/jnj/base/BaseClass.java

```diff
@@ -81,11 +72,10 @@ public static void launchApplication() {
 		header = new HeaderPage();
 		home = new HomePage();
 		difference = new DifferencesPage();
```

Comment:
@zahinSJI initialization of compare and signup is missing

Reply:
They were added, they were not pushed for unknown reason.